### ISSUE TITLE
Rename the default kernel in docker & create intro splash page for 127.0.0.1:8888

### ISSUE THREAD

**Issue by renato-umeton** [Dana-Farber-AIOS/pathml#317]
1. the default kernel created in the docker version should be named pathml or pathmldocker

2. it would be GREAT to have a splash page with Welcome and examples. This is what it looks like now

![image](https://user-images.githubusercontent.com/25375373/174854660-53916464-8aa3-4622-bf4f-350ef840dbba.png)

and this is what it could feature to look like to be more attractive to pathologists:
a. Example gallery
b. Link to documentation
c. Example notebooks from the documentation
d. Link or live preview of info/charts from https://ossinsight.io/analyze/Dana-Farber-AIOS/pathml

**Comment by jacob-rosenthal** [comment]
Agreed. I'm not sure how we can customize the jupyterlab landing page - from [this](https://github.com/jupyterlab/jupyterlab/issues/4382) it looks like maybe it can be done using a 3rd party plugin, but not sure if the added dependency is worth the benefit.

We can certainly copy a tutorial notebook into the working directory though, maybe one for H&E and one for IF?
